src/custom/constants/networks.ts

```diff
@@ -1,30 +1,146 @@
-import { JsonRpcProvider } from '@ethersproject/providers'
+import {
+  Network,
+  Web3Provider,
+  JsonRpcProvider,
+  UrlJsonRpcProvider,
+  StaticJsonRpcProvider,
+  InfuraProvider,
+  CloudflareProvider,
+  AlchemyProvider,
+  PocketProvider,
+  AnkrProvider,
+} from '@ethersproject/providers'
+import * as Sentry from '@sentry/react'
```

Comment:
Fror this `networks` file, i think we should move it to `cow-react` maybe wait until my refactor is done. When we "refactor/centralise" sth, is good we remove it from `custom` and put it in the right place of `cow-react`

Reply:
It is a good point - I'll update in place for now and wait for your refactor to be finished to move them around.